dist/flags.py

```diff
@@ -142,6 +142,7 @@
     'stat' : [
         'STAT_CLEAR',
         'STAT_JSON',
+        'STAT_JSON_PRINT_TABLES',
```

Comment:
@daveh86, I'd suggest instead of creating the `STAT_JSON_PRINT_TABLES` flag, adding a `bool` field `WT_CONNECTION.stat_json_tables` where all the `stat_` fields are defined.  It's functionally the same as the flag, but moving it connects it to those fields.  To partially answer concern raised by @agorrod, there would be a problem if we ever made this stat collection multithreaded, but then `WT_CONNECTION.stat_fs` could not be used as is, if it were multithreaded.  I think grouping together the offenders makes it more obvious.